tapdb/sqlc/migrations/000007_universe.down.sql

```diff
@@ -9,3 +9,5 @@ DROP INDEX IF EXISTS universe_roots_asset_id_idx;
 DROP INDEX IF EXISTS universe_roots_group_key_idx;
 DROP INDEX IF EXISTS universe_leaves_key_idx;
 DROP INDEX IF EXISTS universe_leaves_namespace;
+DROP INDEX IF EXISTS federation_general_sync_config;
```

Comment:
There are no such indexes. Did you mean the tables? If yes, then the names are out of sync.